game/src/plugins/skills/mining/src/gem.kt

```diff
@@ -1,13 +1,18 @@
 package org.apollo.game.plugin.skills.mining
 
-data class Gem(val id: Int, val chance: Int)
-
-private val GEMS = mutableMapOf<Int, Gem>()
+enum class Gem(val id: Int, val chance: Int) {
+    UNCUT_SAPPHIRE(1623, 0), // uncut sapphire
```

Comment:
You can delete these comments - if you're going to add comment, remember to ask yourself "does this add value/meaning to the code?"